src/toffy/qc_comp.py

```diff
@@ -651,38 +652,63 @@ def _compute_qc_tma_metrics(self, tma: str):
                 index=False,
             )
 
-    def qc_tma_metrics_rank(self, tmas: List[str], channel_exclude: List[str] = None):
-        """Creates the average rank for a given TMA across all FOVs and unexcluded channels.
+    def qc_tma_metrics_zscore(self, tmas: List[str], channel_exclude: List[str] = None):
+        """Creates the average zscore for a given TMA across all FOVs and unexcluded channels.
         By default the following channels are excluded: Au, Fe, Na, Ta, Noodle.
 
         Args:
             tmas (List[str]): The FOVs withmetet the TMA in the folder name to gather.
             channel_exclude (List[str], optional): An optional list of channels to further filter
                 out. Defaults to None.
         """
-        with tqdm(total=len(tmas), desc="Computing QC TMA Metric Ranks", unit="TMA") as pbar:
+        max_col, max_row = 0, 0
+        with tqdm(total=len(tmas), desc="Computing QC TMA Metric Z-scores", unit="TMA") as pbar:
             for tma in tmas:
-                self.tma_avg_ranks[tma] = self._compute_qc_tma_metrics_rank(
+                self.tma_avg_zscores[tma] = self._compute_qc_tma_metrics_zscore(
                     tma, channel_exclude=channel_exclude
                 )
+                max_col = (
```

Comment:
Instead of the `if-else` control flow statement, we can use the built in `max` function which returns the largest item given multiple positional arguments. For example `max_col = max(self.tma_avg_zscores[tma].shape[1], max_col)`.